package.json

```diff
@@ -139,6 +143,7 @@
         "googleapis": "^108.0.0",
         "handsontable": "^12.3.3",
         "html-to-text": "^8.2.0",
+        "indefinite": "^2.4.3",
```

Comment:
ah wait, should I rather install this package into the grapher package? (it's only used in Grapher)